lms/djangoapps/verify_student/urls.py

```diff
@@ -35,6 +35,16 @@
         }
     ),
 
+    # This URL is for A/B testing
+    url(
+        r'^upgrade_1/{course}/$'.format(course=settings.COURSE_ID_PATTERN),
```

Comment:
Since we are doing A/B testing so it makes sense to make the suffix like `upgrade_ab/`